src/universal/config/routes.ts

```diff
@@ -42,6 +42,14 @@ export const AppRoutes = {
     '/toeristische-verhuur/vergunning/list/:kind/:page?',
   'TOERISTISCHE_VERHUUR/VERGUNNING':
     '/toeristische-verhuur/vergunning/:casetype/:id',
+  VAREN: '/Passagiers-en-beroepsvaart',
+  'VAREN/LIST': '/Passagiers-en-beroepsvaart/vergunningen/:caseType/:page',
+  'VAREN/DETAIL-EXPLOITATIE':
+    '/Passagiers-en-beroepsvaart/vergunning/varen-vergunning-exploitatie/:id',
+  'VAREN/DETAIL-LIGPLAATS':
+    '/Passagiers-en-beroepsvaart/vergunning/varen-vergunning-ligplaats/:id',
+  'VAREN/DETAIL-BEDRIJFSGEGEVENS':
+    '/Passagiers-en-beroepsvaart/vergunning/reder/:id',
```

Comment:
```suggestion
  VAREN: '/passagiers-en-beroepsvaart',
  'VAREN/LIST': '/passagiers-en-beroepsvaart/vergunningen/:kind/:page?',
  'VAREN/DETAIL':
    '/passagiers-en-beroepsvaart/vergunning/:caseType/:id',
```

Reply:
Het staat in de userstory anders, namelijk:
Thema route /varen ✅

Lijstpagina /varen/vergunningen/:caseType/:page✅

Detail vergunning exploitatie /varen/vergunning/varen-vergunning-exploitatie/:id✅

Detail vergunning ligplaats /varen/vergunning/varen-vergunning-ligplaats/:id✅

Detail bedrijfsgegevens /varen/vergunning/reder/:i

Reply:
Die lijstpagina is een foutje, in mijn comment staat de goede.
Die andere 2 worden door mijn comment gecovered
En de laatste route uit de ticket is outdated.

(Ik heb ticket gemaakt)